config/jobs/kubernetes-sigs/sig-windows/release-1.19-windows.yaml

```diff
@@ -0,0 +1,337 @@
+presubmits:
+  kubernetes/kubernetes:
+  - name: pull-kubernetes-e2e-azure-disk-windows
+    decorate: true
+    always_run: false
+    optional: true
+    run_if_changed: 'azure.*\.go'
+    path_alias: k8s.io/kubernetes
+    branches:
+    - release-1.19
+    labels:
+      preset-service-account: "true"
+      preset-azure-cred: "true"
+      preset-azure-windows: "true"
+      preset-windows-repo-list: "true"
+      preset-k8s-ssh: "true"
+      preset-dind-enabled: "true"
+    extra_refs:
+    - org: kubernetes-sigs
+      repo: azuredisk-csi-driver
+      base_ref: master
+      path_alias: sigs.k8s.io/azuredisk-csi-driver
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20200901-eeeadc5-1.19
+        command:
+        - runner.sh
+        - kubetest
+        args:
+        # Generic e2e test args
+        - --test
+        - --up
+        - --down
+        - --build=quick
+        - --dump=$(ARTIFACTS)
+        # Azure-specific test args
+        - --deployment=aksengine
+        - --provider=skeleton
+        - --aksengine-admin-username=azureuser
+        - --aksengine-admin-password=AdminPassw0rd
+        - --aksengine-creds=$(AZURE_CREDENTIALS)
+        - --aksengine-download-url=https://github.com/Azure/aks-engine/releases/download/v0.55.1/aks-engine-v0.55.1-linux-amd64.tar.gz
```

Comment:
Should we just use the aka.ms redirect (and update it from v0.55.0 to v.0.55.1)?

Reply:
I think we could completely switch to using https://aka.ms/aks-engine/aks-engine-k8s-e2e.tar.gz. I only see a few Azure-related jobs that are using v0.54.1. All other jobs are using v0.55.1. I can open a separate PR for that.

Reply:
@chewong is https://aka.ms/aks-engine/aks-engine-k8s-e2e.tar.gz built automatically from master branch?

Reply:
@feiskyer it is not. 
I've been thinking about setting up something that would build from master nightly that we can use for upstream tests but it hasn't gotten very high on my priority list. Do you think that would be useful for non-Windows tests as well.

Reply:
no, it's versioned and we modify it through aka.ms. We usually change it when there is a new aks-engine release.

Reply:
> @chewong is https://aka.ms/aks-engine/aks-engine-k8s-e2e.tar.gz built automatically from master branch?

no, it's versioned and we modify it through aka.ms. We usually change it when there is a new aks-engine release.

> @feiskyer it is not.
I've been thinking about setting up something that would build from master nightly that we can use for upstream tests but it hasn't gotten very high on my priority list. Do you think that would be useful for non-Windows tests as well.

I think we should keep it versioned for now. Using nightly aks-engine build might introduce another variable that could potentially break the jobs.

Reply:
> Using nightly aks-engine build might introduce another variable that could potentially break the jobs.

+1

Reply:
> Do you think that would be useful for non-Windows tests as well.

Yep, because waiting a new version of aks-engine may be a long time. If it is still versioned, then what's the difference between it and aks-engine releases?

Reply:
These point tot he aks-engine release. The advantage here is only having to update 1 thing (the aks.ms redirect) vs updating all the instance in .yaml :D

Reply:
@marosset good point, thanks! Then it makes sense.